### PR TITLE
Release 3.2.0-BETA.3

### PR DESCRIPTION
**Pull request by kryalama**
Release 3.2.0-BETA.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Log all available jmx metrics at debug level.
+* Misc logging improvements:
+    - Adds PID to startup message. This will help support customers who need to take a thread/heap dump for any reason.
+    - Clearer timezone identifier on logs (-07:00 instead of -07).
+    - add applicationinsights.testing.etw.disabled setting just for use during testing.
```
Comment (heyams) [inline CHANGELOG.md:8]: i think we don't need to mention this? which pr has this change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+* Add grizzly instrumentation as preview.
+* Use tmpdir to store temporary jfr recordings.
```
Comment (heyams) [inline CHANGELOG.md:10]: what is jfr?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+* Fix live metrics sampled counts.
+* Fix stack traces not being captured for deadlocks [#1263](https://github.com/microsoft/ApplicationInsights-Java/issues/1263).
+* Support Sample Telemetry feature to live metrics.
```
Comment (heyams) [inline CHANGELOG.md:13]: please add the link to the pr or the issue.

Comment (heyams) [reply CHANGELOG.md:13]: for live metrics?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+* Add inherited attributes preview [#1743](https://github.com/microsoft/ApplicationInsights-Java/issues/1743)
+* Add App Map to App service/function view support.
```
Comment (heyams) [inline CHANGELOG.md:15]: this is a bug, not a new feature.

Comment (heyams) [reply CHANGELOG.md:15]: please add link too.

File: CHANGELOG.md (hunk 1 above)
Comment (kryalama) [reply CHANGELOG.md:8]: https://github.com/microsoft/ApplicationInsights-Java/pull/1828

File: CHANGELOG.md (hunk 2 above)
Comment (kryalama) [reply CHANGELOG.md:10]: Java flight recorder.  Might be useful for profiler service customers

Comment (heyams) [reply CHANGELOG.md:10]: please be explicit.  or add a link for that.

File: CHANGELOG.md (hunk 1 above)
Comment (heyams) [reply CHANGELOG.md:8]: we don't need to mention 3 changes in that pr.. just link line 5 with that pr.

**Review by heyams — APPROVED** [review]

**Pull request merged**